src/libopensc/gp.c

```diff
@@ -46,25 +46,29 @@ static const struct sc_aid gp_isd_rid = {
 
 /* Select AID */
 int
-gp_select_aid(struct sc_card *card, const struct sc_aid *aid)
+gp_select_aid(struct sc_card *card, const struct sc_aid *aid, u8 *resp, size_t *resplen)
```

Comment:
This looks like an iso7816 command. we already have some of them in opensc.h, maybe it would be better placed there.

Reply:
suggestions?

Reply:
declaration `iso7816_select_aid` in opensc.h with implementation to iso7816.c.
if needed `gp_select_aid(struct sc_card *card, const struct sc_aid *aid)` can be kept here, where the implementation just calls its iso7816 variant (without resp/resplen)